mifosng-android/src/main/res/values/strings.xml

```diff
@@ -160,6 +164,13 @@
     <string name="group_clients">Group Clients</string>
     <string name="identifier_deleted_successfully">Identifier deleted Successfully</string>
     <string name="exact_match">Exact Match</string>
+    <string name="share_charges">Charges</string>
+    <string name="requested_shares">Shares</string>
+    <string name="lockin_period_frequency">Lock In Period Frequency</string>
+    <string name="lockin_period">Lock In Period</string>
+    <string name="min_active_period_frequency">Minimum Active Period Frequency</string>
+    <string name="min_active_period">Minimum Active Period</string>
+    <string name="saving_account">Savings Account</string>
```

Comment:
@mayank-kgp please look this string https://github.com/mayank-kgp/android-client/blob/e3abac9455e661501d555121e08eca5baa7a14eb/mifosng-android/src/main/res/values/strings.xml#L316 It's the same value

Reply:
done.